### PR TITLE
Refactoring small components to vue3

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [ ] Closes #<issue_number>
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by prachi00** [comment]
I had some extra time today so figured it would be nice to start on #4750 #4704 
just refactored really small components, it would be nice to get these out of the way first before moving to bigger ones

File: components/CookieBanner.vue [hunk 1]
```diff
@@ -20,24 +20,17 @@
   </div>
 </template>
 
-<script lang="ts">
-import { Component, Vue } from 'nuxt-property-decorator'
+<script lang="ts" setup>
+let hasDisplayedCookieBanner: boolean =
```
Comment (Jarsen136) [inline components/CookieBanner.vue:24]: It should become a ref value

File: components/TheFooter.vue [hunk 2]
```diff
@@ -75,90 +75,89 @@
...
+    external: true,
+  },
+  {
+    name: $i18n.t('ambassador program'),
+    url: 'https://docs.kodadot.xyz/ambassador-program/ambassador-intro.html#the-opportunity',
+    external: true,
+  },
+]
+
+let socials = [
```
Comment (Jarsen136) [inline components/TheFooter.vue:128]:
```suggestion
const socials = [
```

**Review by Jarsen136 — CHANGES_REQUESTED** [review]
Pls check ths bug.

Kapture 2023-01-20 at 12 29 01

File: components/CookieBanner.vue (hunk 1 above)
Comment (prachi00) [reply components/CookieBanner.vue:24]: oh, fixing.

File: components/bsx/Create/Create.vue [hunk 3]
```diff
@@ -4,18 +4,14 @@
   </div>
 </template>
```
Comment (Jarsen136) [inline components/bsx/Create/Create.vue:6]: Could pls check this GitHub actions error ?

Comment (prachi00) [reply components/bsx/Create/Create.vue:6]: which error?

Comment (Jarsen136) [reply components/bsx/Create/Create.vue:6]: image

Comment (prachi00) [reply components/bsx/Create/Create.vue:6]: oh, let me see

Comment (prachi00) [reply components/bsx/Create/Create.vue]: renamed the component

File: components/bsx/Create/CreateCollectionParent.vue [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<template>
```
Comment (Jarsen136) [inline components/bsx/Create/CreateCollectionParent.vue:1]: TBH the component `CreateCollectionParent` seems a little bit strange : )

File: components/bsx/Asset/AssetTable.vue [hunk 5]
```diff
@@ -48,26 +48,22 @@
...
 
-  @Prop({ type: Array, default: () => emptyArray<AssetItem>() })
-  public assetList!: AssetItem[]
-
-  @Emit('select')
-  tellFrens(id: string) {
-    return id
-  }
+const tellFrens = (id: string) => {
+  emit('select')
```
Comment (Jarsen136) [inline components/bsx/Asset/AssetTable.vue:66]: Have you tested it fully? I guess the data `id ` should be emitted

**Review by Jarsen136 — COMMENTED** [review]
otherwise code lgtm

File: components/bsx/Create/CreateCollectionParent.vue (hunk 4 above)
Comment (prachi00) [reply components/bsx/Create/CreateCollectionParent.vue:1]: what would you rather have it named? createcollection was already there, I couldnt figure out anything better :D

Comment (vikiival) [reply components/bsx/Create/CreateCollectionParent.vue:1]: CreateCollectionPage / CreateCollectionForm

Comment (prachi00) [reply components/bsx/Create/CreateCollectionParent.vue]: changed, thanks

File: components/bsx/Asset/AssetList.vue [hunk 6]
```diff
@@ -86,6 +86,7 @@ const fetchCurrentToken = async (): Promise<void> => {
 }
 
 const handleTokenSelect = async (id: string) => {
+  console.log('idddd', id)
```
Comment (Jarsen136) [inline components/bsx/Asset/AssetList.vue:89]: let's remove this console log

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Review by vikiival — CHANGES_REQUESTED** [review]
There should be no create for Movr and GLRM

**Comment by prachi00** [comment]
> There should be no create for Movr and GLRM

you mean I should delete those components?

**Comment by vikiival** [comment]
Yes,

GLMR and MOVR are read only

https://www.reddit.com/r/KodaDot/comments/w7slmf/comment/ihle9cs/?utm_source=share&utm_medium=web2x&context=3

**Comment by yangwao** [comment]
@vikiival 👀

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
> @vikiival 👀

image

Hello @prachi00 please resolve conflicts

**Comment by prachi00** [comment]
resolved

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 35.02 USD/KSM ~ 1.428 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x695fde11e34345c314141ceda2d83f6c7bf6e78f830715cc763d43cc6b42f1a6](https://kusama.subscan.io/extrinsic/0x695fde11e34345c314141ceda2d83f6c7bf6e78f830715cc763d43cc6b42f1a6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**